tests/unit/TranslateTest.js

```diff
@@ -52,3 +54,38 @@ describe('translate', () => {
         expect(translate.translate('en', ['testKeyGroup', 'testFunction'], {testVariable})).toBe(expectedValue);
     });
 });
+
+describe('Translation Keys', () => {
+    let activeLanguage;
```

Comment:
Why is this "global"? Can't we pass it as a regular argument to `matchKeys`?

Reply:
This is convenient.